### PR TITLE
fix(zk_toolbox): make ecosystem configuration not found error more clear

### PR DESCRIPTION
**Pull request by zk-Lumi**
## What ❔

- Make ecosystem configuration not found error more clear

## Why ❔

- User needs to check readme again to make sure they are in the right folder

## Checklist

- [x] PR title corresponds to the body of PR (we generate changelog entries from PRs).
- [x] Tests for the changes have been added / updated.
- [x] Documentation comments have been added / updated.
- [x] Code has been formatted via `zk fmt` and `zk lint`.
- [x] Spellcheck has been run via `zk spellcheck`.

### CODE REVIEW

File: zk_toolbox/crates/config/src/ecosystem.rs [hunk 1]
```diff
@@ -201,7 +201,7 @@ impl EcosystemConfig {
 /// Result of checking if the ecosystem exists.
 #[derive(Error, Debug)]
 pub enum EcosystemConfigFromFileError {
-    #[error("Ecosystem configuration not found")]
+    #[error("Ecosystem configuration not found (make sure you have created one `zk_inception ecosystem create` + are in the ecosystem folder `cd path/to/ecosystem/name`)")]
```
[the remark this answers is not in the document]
Comment (zk-Lumi) [reply zk_toolbox/crates/config/src/ecosystem.rs:204]: Most common I see for this error is when people forget to cd into ecosystem folder after `zk_inception ecosystem create`. Can you think of alternative wording you would prefer to my wording?

[the remark this answers is not in the document]
Comment (Deniallugo) [reply zk_toolbox/crates/config/src/ecosystem.rs:204]: Probably at the end of the create phase, we could show that you have to cd to the new path?

[the remark this answers is not in the document]
Comment (zk-Lumi) [reply zk_toolbox/crates/config/src/ecosystem.rs:204]: Thats a good idea also but I think most people will overlook that, would be good to have it in the error too

[the remark this answers is not in the document]
Comment (zk-Lumi) [reply zk_toolbox/crates/config/src/ecosystem.rs]: @Deniallugo updated wording + added message in ecosystem created, mind reviewing?

File: zk_toolbox/crates/config/src/ecosystem.rs [hunk 2]
```diff
@@ -214,7 +214,7 @@ impl EcosystemConfig {
 /// Result of checking if the ecosystem exists.
 #[derive(Error, Debug)]
 pub enum EcosystemConfigFromFileError {
-    #[error("Ecosystem configuration not found")]
+    #[error("Ecosystem configuration not found (Could not find 'ZkStack.toml' in <path>: Make sure you have created an ecosystem & are in the new folder `cd path/to/ecosystem/name`)")]
```
Comment (Deniallugo) [inline zk_toolbox/crates/config/src/ecosystem.rs:217]: does it make sense to print the actual path? or no?

Comment (zk-Lumi) [reply zk_toolbox/crates/config/src/ecosystem.rs]: makes sense to me - Updated

**Comment by Deniallugo** [comment]
closed in favour of #2424

**Pull request closed without merging**